### DISCUSSION TITLE
texture compression tools

### DISCUSSION THREAD

**Discussion by shibeiji20** [openscenegraph/OpenSceneGraph#1279] [Q&A]
Are there some open-source texture compression tools supported by OSG? NVTT3 is used in my project currently, and I found that the compression ratio is acceptable, but the compression efficiency is much slow, expecially in lod. 
Any suggestion?

**Comment by xarray** [comment]
Maybe you could have a look at osgverse (https://github.com/xarray/osgverse). It supports BASISU compression with libKTX dependency and can give a much better ratio on disk, and quickly transcoding to compression format like DDS and ETC.

**Reply by shibeiji20** [reply]
OK , thanks very much, I'll try that
